README.md

```diff
@@ -7,9 +7,17 @@ Click the button to run the tutorial:
 
 [![Open in Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.png)](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/jhipster/jhipster-guides&tutorial=guides/00_setting_up_your_environment.md)
 
-Please note that clicking this button will open the Google Cloud Shell in a new tab, clone the repository tutorials and start the tutorial n°0.
+**Please note that clicking this button will open the Google Cloud Shell in a new tab, clone the repository tutorials and start the tutorial n°0**
 
-To run a particuliar tutorial, use this command in the Google Cloud Shell, replacing  `YOUR_TUTORIAL_NAME` by the name of the tutorial that you want to do:
+We highly recommend you to follow the tutorial in the right order. By following these tutorials, you will create an application named BugTrackerJHipster that is inspired from the famous Ticket Issue system from GitHub.
+
+Google Cloud Shell allows you to display markdown files to act as a tutorial and lends you a shell console hosted on the cloud.
+
+It is possible to follow the tutorials on your local machine, but be aware that the `teachme` and the `cloudshell launch-tutorial -d` commands are meant to be used only for Google Cloud Shell.
+
+It should be noted that the tutorials are made to be used in the first place with Google Cloud Shell, that is why we are using absolute path (~), if you are doing this tutorial on your local machine, you can freely choose the directory in which you will generate the application.
+
+To run a particular tutorial, use this command in the Google Cloud Shell, replacing  `YOUR_TUTORIAL_NAME` by the name of the tutorial that you want to do:
```

Comment:
Maybe it would be better to do a list liek 
```bash
teachme ~/jhipster-guides/guides/tuto1.md
```
```bash
teachme ~/jhipster-guides/guides/tuto2.md
```
This way people can click the command so that it is copied to the cloud shell terminal.

Reply:
I think it is good idea, but to click on the command would mean that you are already on the readme.md, and to access it you have no choice but to use the command at least once. At this point, it is counter productive to:
1. Go to the readme.md file
2. Copy the command and press enter

instead of

1. Write the command and press enter

At this moment, at the end of each tutorial, there is a command to go to the next tutorial but maybe it would be better to put the list down there so they can navigate freely?

Reply:
I don't know you decide !